broken_link_checker/checker.py

```diff
@@ -73,6 +74,12 @@ def __init__(self, host: str, delay: int = 1):
             re.IGNORECASE
         )
 
+        # We build a dict of HTTPStatus code
+        # We want more descriptive http status
+        self.http_status = {}
+        for http_status in HTTPStatus:
+            self.http_status[http_status.value] = http_status.description
```

Comment:
Am understanding the need of doing his but i think it's a bottleneck we don't need actually 🤔 
it will loop on all httpstatus each time we will create an instacne of the checker and this is not optimal at all ! 🤔 

Second point : is it really possible to loop on the `HTTPStatus` object ?
For what i can see in the documentation, it looks different... so i may ask.... is this piece of code actually works ?

cf : https://docs.python.org/3/library/http.html

Reply:
It's true, i will remove this part. What do you think to do it while the building of the report?

Reply:
good idea too !